### PR TITLE
[EASY] add badges to readme

### PR DESCRIPTION
**Pull request by seve**
I thought it would be a nice touch to add a few more badges to the readme

Changes:

- In `README.md`
   - Updated DOI badge to zenodo's latest badge
   - Added latest deployment version on PyPi with a link to PyPi
   - Added number of PyPi monthly downloads with link to pypistats.org
   - Added GitHub activity stats to show repo's freshness

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 > an interactive explorer for single-cell transcriptomics data
 
-[![DOI](https://zenodo.org/badge/105615409.svg)](https://zenodo.org/badge/latestdoi/105615409)
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3338548.svg)](https://doi.org/10.5281/zenodo.3338548)
```
Comment (neuromusic) [inline README.md]: Why the change here? This is a static DOI for the 0.11.0 release. The current badge will automatically update the badge and link as we cut new releases.

Comment (seve) [reply README.md]: Not sure on the reasoning but the provided badge on https://zenodo.org/record/3235021#.XO8extNKjBI has changed to my diff

Comment (neuromusic) [reply README.md]: more context on how zenodo handles DOI versioning: http://help.zenodo.org/#versioning

File: README.md [hunk 2]
```diff
@@ -4,6 +4,10 @@
 
 [![DOI](https://zenodo.org/badge/105615409.svg)](https://zenodo.org/badge/latestdoi/105615409)
 
+[![PyPI](https://img.shields.io/pypi/v/cellxgene)](https://pypi.org/project/cellxgene/) [![PyPI - Downloads](https://img.shields.io/pypi/dm/cellxgene)](https://pypistats.org/packages/cellxgene)
```
Comment (neuromusic) [inline README.md]:
```suggestion
[![PyPI](https://img.shields.io/pypi/v/cellxgene)](https://pypi.org/project/cellxgene/) 

[![PyPI - Downloads](https://img.shields.io/pypi/dm/cellxgene)](https://pypistats.org/packages/cellxgene)
```
